src/crawlee/playwright_crawler/_playwright_crawler.py

```diff
@@ -48,6 +49,7 @@ def __init__(
         browser_pool: BrowserPool | None = None,
         browser_type: BrowserType | None = None,
         headless: bool | None = None,
+        pre_navigation_hooks: List[PlaywrightHook] | None = None,
```

Comment:
This is a perfectly valid way of passing the hooks, but have you considered adding a decorator instead/as well? We did a similar change for request handlers because Python does not have anonymous functions (that can contain more than one statement).